README.md

```diff
@@ -12,13 +12,15 @@ Set up a local development environment for COMIT apps with one command.
 
 ## 2 - Create your first project!
 
-1. Create [hello-swap](https://github.com/comit-network/hello-swap/) app: `yarn create comit-app new hello-swap`,
-2. Start environment (blockchain nodes and COMIT nodes): `yarn create comit-app start-env`
-3. Run hello-swap (in separate terminal):
-    1. Navigate to the example app directory `cd <path-to-hello-swap>`
+1. `yarn create comit-app new <your-app-name>`,
+2. `yarn create comit-app start-env` (start blockchain nodes and COMIT nodes)
+3. Run the [btc-eth](https://github.com/comit-network/create-comit-app/tree/master/new_project/examples/btc-eth) example (in separate terminal):
+    1. Navigate to the btc-eth example app directory `cd <path-to-your-app>/examples/btc-eth`
     2. `yarn install` (or `npm install`) to install dependencies
```

Comment:
We only ship a `yarn.lock` file. That doesn't work with `npm`. We should either ship a `package-lock.json` file or limit our instructions to `yarn`.

Reply:
since the description including `npx` adds unnecessary complexity I decided to remove the whole `npm`/`npx` part.